### PR TITLE
DOC-1490: Updated doc for default settings

### PR DESCRIPTION
**Pull request by tiny-ben-tran**
Related Ticket:  DOC-1490

Description of Changes:
- Updated the default value of `a11ycheck_html_version`, `element_format`, `table_style_by_css`, `link_default_protocol` and `table_use_colgroups` settings. 
- Changed the wording in a few places

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~`_data/nav.yml` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by EkimChau — APPROVED** [review]

File: modules/ROOT/partials/configuration/table_style_by_css.adoc [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [[table_style_by_css]]
 == `+table_style_by_css+`
 
-This option enables you to force the Table Properties dialog to use HTML5/CSS3 standards for setting cell spacing and cellpadding. By default, these are added as attributes to the table element. By setting this to true, cell spacing is applied to the table element as a `+border-spacing+` style and cell padding is applied to all `+td+` elements as a `+padding+` style.
+This option enables you to force the Table Properties dialog to use HTML5/CSS3 standards for setting cell spacing and cellpadding. By default, cell spacing is applied to the table element as a `+border-spacing+` style and cell padding is applied to all `+td+` elements as a `+padding+` style. By setting this to false, these are added as attributes to the table element.
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/table_style_by_css.adoc:4]:
```suggestion
This option enables you to force the Table Properties dialog to use HTML5/CSS3 standards for setting cell spacing and cellpadding. By default, cell spacing is applied to the table element as a `+border-spacing+` style and cell padding is applied to all `+td+` elements as a `+padding+` style. By setting this to `false`, these are added as attributes to the table element.
```

**Review by lnewson — APPROVED** [review]

**Comment by lnewson** [comment]
@tiny-ben-tran looks like the `link_default_protocol` default was missed here as it's still using `http` when it should be `https`.

- [ ] Fix `link_default_protocol` default

File: modules/ROOT/partials/configuration/element_format.adoc [hunk 2]
```diff
@@ -1,11 +1,11 @@
 [[element_format]]
 == `+element_format+`
 
-This option controls whether elements are output in the HTML or XHTML mode. `+xhtml+` is the default state for this option. This means that for example `+<br />+` will be `+<br>+` if you set this option to `+html+`.
+This option controls whether elements are output in the HTML or XHTML mode. `+html+` is the default state for this option. This means that for example `+<br />+` will be `+<br>+` by default.
 
 Type : `+String+`
 
-Default Value : `+'xhtml'+`
+Default Value : `+'html'+`
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/element_format.adoc:8]: The example below here needs updating to show something that's not the default.

File: modules/ROOT/partials/configuration/a11ychecker_html_version.adoc [hunk 3]
```diff
@@ -6,7 +6,7 @@ For example: Setting the version to HTML 4 will enable the rule "Complex tables
 
 Type : `+String+`
 
-Default value : `+html4+`
+Default value : `+html5+`
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/a11ychecker_html_version.adoc:9]: The example below here needs updating to show something that's not the default.

**Pull request merged**